kimchi/src/circuits/witness/sum_copy_bits_cell.rs

```diff
@@ -0,0 +1,40 @@
+use ark_ff::Field;
```

Comment:
Reminder.  This is part of this PR or from XOR and not used anymore, so should be removed.